kale/embed/supergraph.py

```diff
@@ -0,0 +1,166 @@
+from typing import List
+
+import networkx as nx
+import torch
+
+
+class SuperVertex(object):
+    def __init__(
+        self,
+        name: str,
+        node_feat: torch.Tensor,
+        edge_index: torch.Tensor,
+        edge_type: torch.Tensor = None,
+        edge_weight: torch.Tensor = None,
+    ) -> None:
+        r"""
```

Comment:
This is a wrong place for class docs. You need to browse through a few more other APIs in PyKale to learn, including https://github.com/pykale/pykale/blob/main/kale/embed/gripnet.py, to understand the difference.